lib/app/views/application/_nav.erb

```diff
@@ -26,11 +26,13 @@
   </ul>
 
   <ul class="nav navbar-nav navbar-right">
-    <% if !current_user.guest? %>
-      <%= erb :"application/nav/five_a_day" %>  
-      <% if current_user.sees_exercises? %>
-        <li><a href="/inbox">Exercises</a></li>
-      <% end %>
+    <% if current_user.show_five? == true %>
```

Comment:
This comparison is the equivalent of `true == true` or `false == true`. Do you see how it can be simplified?